.gitignore

```diff
@@ -0,0 +1,8 @@
+build
+linux-4.9
+media
+toolchain
+mkapp/image
+mkapp/ota_app
```

Comment:
we probably want to track (and delete from the repository) `mkapp/app/app/HDZGOGGLE` here too

Reply:
seems the binary got modified, not removed.

Reply:
Yes, fixed and force pushed